### PR TITLE
Like to help modal

### PR DESCRIPTION
**Pull request by nicholasmorgan**

### CODE REVIEW

**Review by matejoslav — CHANGES_REQUESTED** [review]
Modal opens everytime you load the page.

Tests are missing to test that clicking the button opens the modal and clicking close button closes the modal.

image
The X appears low resolution, we should get .svg file for this.

Would be nice if clicking outside of the modal closed it.

Modal itself looks nice.

File: web-app/src/components/HelpSection.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 import styled from "styled-components";
 import Card from "./Card";
+import LikeToHelpModal from "./LikeToHelpModal";
 import { Routes } from "../App.js";
 import { ReactComponent as Illustration3 } from "../assets/images/Illustration3.svg";
 import { ReactComponent as Illustration4 } from "../assets/images/Illustration4.svg";
 import devices from "../styles/Devices";
 
 const HelpSection = () => {
+  const [likeToHelpModalVisible, setLikeToHelpModalVisible] = useState(true);
```
Comment (matejoslav) [inline web-app/src/components/HelpSection.js:11]: Should be false by default I think

Comment (nicholasmorgan) [reply web-app/src/components/HelpSection.js]: updated

**Comment by nicholasmorgan** [comment]
Added the above

**Comment by nicholasmorgan** [comment]
Regarding adding being able to close the modal by clicking out of it, I think that should be added as a separate ticket

**Comment by MadelinTom** [comment]
Looks good!

**Review by matejoslav — APPROVED** [review]

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210112-120714 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210112-120714") to [dev](https://console.bluemix.net/apps/b98a4dd3-7816-44c4-9933-d50225ef97ee?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 2af43830c2462dc51a2bdce552512168b102b851

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210119-174451 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210119-174451") to [test](https://console.bluemix.net/apps/ffe68851-6e56-40b0-80a3-554fcbfafe9d?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:test"), including commit 2af43830c2462dc51a2bdce552512168b102b851

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210205-152747 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210205-152747") to [test](https://console.bluemix.net/apps/1d38058e-0941-478d-ab73-1c5c09e8e146?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:test"), including commit 2af43830c2462dc51a2bdce552512168b102b851

**Pull request merged**